### DISCUSSION TITLE
Clearer documentation of and better insight into card configuration

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#53] [Feature Requests]
I'm just starting to explore using the cards tool for our documentation set.

The configuration options are a little difficult to navigate. After being extremely frustrated that I couldn't see the link that I had set for a card, I finally found it when I clicked **Show hidden columns**. The card link is not really a column, and should be more immediately visible. And the display elements within a card also are not really "columns" in the traditional sense.

The documentation for the cards tool is incomplete. Many of the options do not have any explanation. For example:

- How many elements can you add to a card?
- What types of elements can you add?
- What is a cover image? How does it display for each size of card?
- There is a **Show labels** option that is enabled by default, but I can't see any difference when I toggle it on and off. What does that do?

**Comment by taszap** [comment]
Hi @JaniceManwiller,

Firstly, I wanted to thank you for thoughtful post! This is a very helpful feedback which I have now tracked internally as an improvement to our docs. 

It also led us to creating a new label in the community 'documentation opportunities' where I will move your post as we would like to encourage more users to share their observations/suggestions. 

Do you still require assistance with the cards? If so, we will start a support ticket with you while we work on updating the docs.

**Comment by JaniceManwiller** [comment]
Thanks Natasza,

I think I'm all right for the time being. I was able to set up the cards
with some trial and error - currently just experimenting with the idea of
linked cards containing a couple of lines of text - but just felt like
there wasn't a lot of information to work from.

Thanks for the quick response!

Janice

**Comment by ChucKN0risK** [comment]
I agree with @JaniceManwiller. First time I used Cards I had a 3rd text column I didn't want and I couldn't find how to remove it. Cards display settings were hard to find.
